storm/metadata.csv

```diff
@@ -1,95 +1,95 @@
-storm.topologyStats.metrics.spouts.last_60.acked,gauge,,sample,,Number of Tuples Acked by Spout & Stream,1,storm,
-storm.topologyStats.metrics.spouts.last_60.complete_ms_avg,gauge,,millisecond,,Complete Tuple Latency by Spout & Stream,-1,storm,
-storm.topologyStats.metrics.spouts.last_60.emitted,gauge,,sample,,Number of Tuples Emitted by Spout & Stream,1,storm,
-storm.topologyStats.metrics.spouts.last_60.executed,gauge,,sample,,Number of Tuples Executed by Spout & Stream,1,storm,
-storm.topologyStats.metrics.spouts.last_60.executed_ms_avg,gauge,,millisecond,,Execute Tuple Latency by Spout & Stream,-1,storm,
-storm.topologyStats.metrics.spouts.last_60.failed,gauge,,sample,,Number of Tuples Failed by Spout & Stream,-1,storm,
-storm.topologyStats.metrics.spouts.last_60.process_ms_avg,gauge,,millisecond,,Process Tuple Latency by Spout & Stream,-1,storm,
-storm.topologyStats.metrics.spouts.last_60.transferred,gauge,,sample,,Number of Tuples Transferred by Spout & Stream,1,storm,
-storm.worker.last_60.assignedCpu,gauge,,percent,core,Assigned Worker CPU Percentage,0,storm,
-storm.worker.last_60.assignedMemOffHeap,gauge,,mebibyte,,Off Heap Memory Assigned for a Worker,0,storm,
-storm.worker.last_60.assignedMemOnHeap,gauge,,mebibyte,,On Heap Memory Assigned for a Worker,0,storm,
-storm.worker.last_60.componentNumTasks,gauge,,task,task,Total Number of Component Tasks for a Worker,0,storm,
-storm.worker.last_60.executorsTotal,gauge,,thread,,Total Number of Executors for a Worker,0,storm,
-storm.worker.last_60.uptimeSeconds,gauge,,second,,Worker Uptime,1,storm,
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+storm.bolt.last_60.acked,gauge,,sample,,Number of Acked Tuples,1,storm,,
+storm.bolt.last_60.capacity,gauge,,fraction,,Bolt Capacity,-1,storm,,
+storm.bolt.last_60.emitted,gauge,,sample,,Number of Emitted Tuples,1,storm,,
+storm.bolt.last_60.errorLapsedSecs,gauge,,second,,Number of Seconds Since Last Error,1,storm,,
+storm.bolt.last_60.executed,gauge,,sample,,Number of Tuples Executed,1,storm,,
+storm.bolt.last_60.executeLatency,gauge,,millisecond,,Bolt Execute Latency,-1,storm,,
+storm.bolt.last_60.executors,gauge,,thread,,Number of Bolt Executors,0,storm,,
+storm.bolt.last_60.failed,gauge,,sample,,Number of Failed Tuples,-1,storm,,
+storm.bolt.last_60.processLatency,gauge,,millisecond,,Bolt Process Latency,-1,storm,,
+storm.bolt.last_60.requestedCpu,gauge,,percent,core,Bolt Requested CPU,0,storm,,
+storm.bolt.last_60.requestedMemOffHeap,gauge,,mebibyte,,Bolt Requested Memory Off Heap,0,storm,,
+storm.bolt.last_60.requestedMemOnHeap,gauge,,mebibyte,,Bolt Requested Memory On Heap,0,storm,,
+storm.bolt.last_60.tasks,gauge,,task,task,Bolt Tasks,0,storm,,
+storm.bolt.last_60.transferred,gauge,,sample,,Number of Transferred Tuples,1,storm,,
+storm.cluster.availCpu,gauge,,core,,Available Storm Cluster CPU,0,storm,,
```

Comment:
given these have cpu && mem in the name, should these be curated?

Reply:
So the idea now is to add the column to all csv files to keep the files consistent. But as of now we probably wont add curation values for this integration in v1. [These are the integrations that we will likely start with when GAing](https://datadoghq.atlassian.net/wiki/spaces/PROC/pages/2352416524/Selecting+Curated+Metrics)

But happy to add your recommended curated metrics for this integration in a followup iteration!